src/transformers/pipelines/image_text_to_text.py

```diff
@@ -250,21 +260,22 @@ def __call__(
         Generate a text given text and the image(s) passed as inputs.
 
         Args:
-            images (`str`, `List[str]`, `PIL.Image or `List[PIL.Image]`):
+            images (`str`, `List[str]`, `PIL.Image, `List[PIL.Image]`, `List[Dict[str, Union[str, PIL.Image]]]`):
                 The pipeline handles three types of images:
 
                 - A string containing a HTTP(s) link pointing to an image
                 - A string containing a local path to an image
                 - An image loaded in PIL directly
 
-                The pipeline accepts either a single image or a batch of images.
+                The pipeline accepts either a single image or a batch of images. Finally, this pipeline also supports
+                the chat format (see `text`) containing images and text in this argument.
```

Comment:
Not the prettiest solution, as technically `images` can now contain a chat with 100% text.

The alternative was to absorb all positional arguments with `*`, e.g. 
```py
def __call__(
        self,
        *images_or_text,
        images=None,
        text=None,
        **kwargs
)
```

and add the logic for all possible cases, but that seemed... overly complex?

Reply:
Fine for me as it is now